velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -431,5 +431,19 @@ TEST_F(DateTimeFunctionsTest, addMonths) {
       addMonths("2023-07-10", kMax),
       fmt::format("Integer overflow in add_months(2023-07-10, {})", kMax));
 }
+
+TEST_F(DateTimeFunctionsTest, monthDate) {
+  const auto month = [&](std::optional<int32_t> date) {
```

Comment:
It may be better to let this lambda accept a date string and then parse it to date type internally, which may be more friendly to reader. See reference:
https://github.com/facebookincubator/velox/blob/e3d28a16c0a08297f279f71efb76d07e2552f264/velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp#L407

Reply:
@PHILO-HE Updated.